16 Documentation for build and publish process of docker image. (#231)
commercetools/commercetools-project-sync: docs/CONTRIBUTING.md

````

### Build 

 Gradle docker plugin is used to build and deploy the docker images. 

>  To build the docker image to your desktop docker,

Reviewer: ```suggestion
 To build the docker image locally, please run
```

 ````bash
./gradlew dockerBuildImage
````
The docker image has been built and published to your desktop docker.

Example:

> For testing all resources sync,

Reviewer: ```suggestion
For testing all resources, please run:
```

````bash
docker run commercetools/commercetools-project-sync:<version> -s all
````

For more detailed information on the build and release process, see [Build and Release](BUILD.md) documentation.

## License

> Copyright (c) 2019 commercetools

Reviewer: ```suggestion
Copyright (c) 2021 commercetools
```

> Copyright (c) 2021 commercetools

Reviewer 2: not sure but copyright statement year might be better if you don't touch it? 🤔 
see: https://stackoverflow.com/a/2391555

Author: Yes, I think it is better if we don't touch copyright statement year.
@ahalberkamp do you agree?

Reviewer: yes leave it untouched

Author: https://github.com/commercetools/commercetools-project-sync/pull/231/commits/bba622f7259e3ca338c0e8612c304f37bc32bdc2
